iw/filter_gui.py

```diff
@@ -130,6 +130,24 @@ def __init__(self, xml):
     def addToUI(self, tuple):
         pass
 
+    def incrementDeviceCount(self, tuple):
```

Comment:
Could you please change the ``tuple`` name to something more self explanatory and also add brief docstring for this method.